src/main/resources/db/migration/dbchangelog.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <changeSet  id="1"  author="Max">
+        <createTable  tableName="user">
+            <column  name="id"  type="int">
+                <constraints  primaryKey="true"  nullable="false"/>
+            </column>
+            <column  name="firstname"  type="varchar(50)"/>
+            <column  name="lastname"  type="varchar(50)"/>
+            <column  name="email"  type="varchar(50)"/>
+            <column  name="phonenumber"  type="varchar(50)"/>
+        </createTable>
+    </changeSet>
+    <changeSet id="2" author="Max">
+        <createTable tableName="order">
+            <column  name="id"  type="int">
+                <constraints  primaryKey="true"  nullable="false"/>
+            </column>
+            <column  name="user_id"  type="int">
+                <constraints foreignKeyName="fk_order_user_id" references="user(id)"/>
+            </column>
+            <column name="order_date" type="varchar(50)"/>
+            <column name="status" type="varchar(50)"/>
+        </createTable>
+    </changeSet>
+    <changeSet id="3" author="Max">
+        <createTable tableName="rating">
+            <column name="id" type="int">
+                <constraints  primaryKey="true"  nullable="false"/>
+            </column>
+            <column name="rating" type="varchar(50)"/>
+        </createTable>
+    </changeSet>
+    <changeSet id="4" author="Max">
+        <createTable tableName="product">
+            <column name="id" type="int">
+                <constraints  primaryKey="true"  nullable="false"/>
+            </column>
+            <column name="name" type="varchar(50)"/>
+            <column name="price" type="varchar(50)"/>
+            <column name="creationdate" type="varchar(50)"/>
+            <column name="rating_id" type="int">
+                <constraints foreignKeyName="fk_product_rating_id" references="rating(id)"/>
```

Comment:
It's incorrect reference. product should not know anything about it's rating. Please refer to another comment: https://github.com/maxturko/MaxDan/pull/1/files#r564376729